js/web/lib/onnxjs/graph.ts

```diff
@@ -585,33 +585,48 @@ class GraphImpl implements Graph, Graph.Transformer {
   finalizeGraph() {
     let offset = 0;
     // delete all nodes that are not being executed
+    // The graph is represented using these two arrays
+    // this._nodes - Array holding the kernels to execute - each entry is a kernel pointing to this._allData
+    // this._allData - hold 2 fields - to [] & from [] - these arrays hold the graph map for inputs and outputs per node
+    // reducedMap - remapping the graph after reading the flag 'executeNode' 
```

Comment:
`reducedMap` in comments and `reducedGraph` in code is not a good name to describe itself. Suggest to name it `newIndices`